### ISSUE TITLE
More global styling options, especially for breakpoints.

### ISSUE THREAD

**Issue by pascalinger** [soflyy/oxygen-bugs-and-features#1536]
**Describe the feature you'd like to see included in Oxygen.**
I would love to see more options for global styling. I will just detail a few examples of stuff I would like to be able to change globally.

- Text Sizes on different breakpoints
- Default column padding in columns element (also breakpoint options)
Let me just say that this one has been the most frustrating for me. The padding for columns is set for the id of the column, thus a simple class override is not possible unless I use !important. I would love it if that css rule could just be replaced.
- Section spacing per breakpoint
- Ability to rename global colors
- margins for body text elements (p) (paragraph spacing, potentially with a :not(:last-of-type))

**Comment by pascalinger** [comment]
For anyone curious how I change the padding for columns globally at the moment (not happy about the !important there): [id^=new_columns-] > div {
  padding: 16px !important;
}

**Comment by pascalinger** [comment]
**Update** The new version of Oxygen 3.6 allows us to modify the column spacing. If at some point we get breakpoint options in there as well, I'll be really happy.

**Comment by nickaharding** [comment]
I agree, global settings for different breakpoints (Heading font sizes, spacing etc.) is critical to my workflow and one if the only features I've noticed that really lacks behind other builders.
